test.md

```diff
@@ -131,12 +132,13 @@ This asserts related operation about tables.
 ```k
     syntax Assertion ::= "#assertEmptyTable"    Int MaxBound String
                        | "#assertEmptyTableAux" Int Int MaxBound String
- // --------------------------------------------------------------------
+ // -------------------------------------------------------------------
     rule <k> #assertEmptyTable SIZE MAX MSG => #assertEmptyTableAux (NEXT -Int 1) SIZE MAX MSG ... </k>
          <nextTabAddr> NEXT </nextTabAddr>
 
     rule <k> #assertEmptyTableAux ADDR SIZE MAX _ => .  ... </k>
-         <tabAddrs> (0 |-> ADDR) => .Map </tabAddrs>
+         <nextTabIdx> NEXT => NEXT -Int 1 </nextTabIdx>
+         <tabIndices> ... ( 0 |-> ADDR ) => .Map ... </tabIndices>
```

Comment:
The `<tabIndinces>` should only ever have a single element for now right? Do we need the `...`?

Reply:
No. I fixed it